<?xml version="1.0" encoding="UTF-8"?>
<!--
     Okno s uzivatelskymi nastaveniami.
     Autori:
       Lisik Miroslav <xlisik00@stud.fit.vutbr.cz>
       Pokorny Fridolin <fridex.devel@gmail.com>
-->
<ui version="4.0">
 <class>preferencesWindow</class>
 <widget class="QDialog" name="preferencesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Petri net item style</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="arrowLabel">
            <property name="text">
             <string>Arrow style:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="transitionLabel">
            <property name="text">
             <string>Transition style:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="placeLabel">
            <property name="text">
             <string>Place style:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="arrowStyleBox">
            <item>
             <property name="text">
              <string>Classic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Colored</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold colored</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="transitionStyleBox">
            <item>
             <property name="text">
              <string>Classic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Colored</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold colored</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="placeStyleBox">
            <item>
             <property name="text">
              <string>Classic</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Colored</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bold colored</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Font settings</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="fontLabel">
            <property name="text">
             <string>Font:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QFontComboBox" name="fontComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Default connection settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="hostLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>330</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Host:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="userLabel">
            <property name="text">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="passwordLabel">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="1">
           <widget class="QCheckBox" name="passView">
            <property name="text">
             <string>Hide</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="portBox">
            <property name="maximum">
             <number>65535</number>
            </property>
            <property name="value">
             <number>44444</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="hostEdit">
            <property name="text">
             <string>localhost</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLineEdit" name="userEdit">
            <property name="text">
             <string>user</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLineEdit" name="passwordEdit">
            <property name="text">
             <string>user</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferencesWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferencesWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
